### PR TITLE
clarify priority between master flag and kubeconfig

### PR DESCRIPTION
**Pull request by kadisi**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind documentation
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/setup/setup.md [hunk 1]
```diff
@@ -26,9 +26,16 @@
...
+      master: http://localhost:8080
       ...
       kubeconfig: "path_to_kubeconfig_file" #Enter path to kubeconfig file to enable https connection to k8s apiserver
+  devicecontroller:
+    kube:
+      master: http://localhost:8080
+      ...
+      kubeconfig: "path_to_kubeconfig_file" #Enter path to kubeconfig file to enable https connection to k8s apiserver 
   ```
+but if `master` and `kubeconfig` are both setted, `master` will overrides any value in kubeconfig.
```
Comment (kevin-wangzefeng) [inline docs/setup/setup.md]: s/setted/set/g

Comment (kevin-wangzefeng) [reply docs/setup/setup.md]: s/will overrides/will override/

Comment (kadisi) [reply docs/setup/setup.md]: ok

File: docs/setup/setup.md [hunk 2]
```diff
@@ -26,9 +26,16 @@
   ```yaml
```
Comment (fisherxu) [inline docs/setup/setup.md]: Enter the **master address** or path to kubeconfig file in controller.yaml

Comment (kadisi) [reply docs/setup/setup.md]: ok

**Comment by fisherxu** [comment]
Seems the e2e will failed, if we use the kubeconfig default.

**Comment by fisherxu** [comment]
#856 change the e2e to dynamic, can taken a look :) @kadisi

**Comment by kadisi** [comment]
@fisherxu  has fixed,     edgecore run error when can not find kubeconfig file,  so i delete default kubeconfig path in e2e test.

**Comment by kadisi** [comment]
@fisherxu  has fixed,     edgecore run error when can not find kubeconfig file,  so i delete default kubeconfig path in e2e test.

**Comment by sids-b** [comment]
Thanks @kadisi 

/lgtm

**Comment by fisherxu** [comment]
/lgtm
Thanks!

**Pull request merged**